src/002_installation.md

```diff
@@ -78,25 +78,68 @@ Continue as described in Section [Running Tamarin](#sec:running-tamarin) to run
 Mac OS X {#sec:macosx}
 --------
 
-### Installing the pre-built Tamarin binary {#sec:MacOSBinInstall}
+### Automatic Installation of the Tamarin binary {#sec:MacOSBinInstall}
 
-To run Tamarin on Mac OS X you need to have Maude 2.7 and GraphViz. If you have [HomeBrew](http://brew.sh) installed, you can simply 
-`brew install homebrew/science/maude graphviz` to satisfy the pre-requisites of running the binary. Then follow from step 3 onwards.
+The fastest way to install Tamarin on Mac OS X is to use [Homebrew](http://brew.sh/) [Cask](https://caskroom.github.io/).
 
-Otherwise, to install Maude and GraphViz manually:
+If you already have this installed, it is as simple as running the following in your terminal:
+
+  * `brew cask install tamarin-prover`
```

Comment:
Does this work if Maude is not on your PATH? When we tested it both you and I already had maude on our PATH for tamarin to pick up so not sure it will work on a clean install